frontend/templates/views/partials/banner.j2

```diff
@@ -1,7 +1,23 @@
 {% do doc.styles.addCssFile('/css/components/molecules/banner.css') %}
-
+<!--
```

Comment:
Do you need to comment this out or can you just delete it? (it's a git revert away)

Reply:
You're right. I just didn't feel good about removing it :)